pyblazing/pyblazing/apiv2/context.py

```diff
@@ -2193,6 +2193,9 @@ def create_table(self, table_name, input, **kwargs):
                     row_groups_ids.append(row_group_ids)
                 table.row_groups_ids = row_groups_ids
 
+            # want all `column_names` be a list of strings
+            table.column_names = [i.decode() for i in table.column_names]
```

Comment:
This should be handled at the source. Where is column_names being set originally? Please fix this at the source.